### ISSUE TITLE
The training script for tiny-imagenet not working

### ISSUE THREAD

**Issue by Linwei94** [torrvision/focal_calibration#17]
The training script for tiny-imagenet not working. The dataset setting in train.py is can not load the validation set as the paper says.
